src/matcha_ml/services/analytics_service.py

```diff
@@ -30,10 +30,44 @@ class AnalyticsEvent(str, Enum):
     GET = "get"
 
 
-def execute_analytics_event(
+def _get_state_uuid() -> Optional[str]:
```

Comment:
This should return the `MatchaResourceProperty` that's brought back by the `find_property()` function call. Returning a `str` is ambiguous

Reply:
Changed 👍